### ISSUE TITLE
Specify minor version in information endpoint

### ISSUE THREAD

**Issue by daradermody** [regel/loudml#73]
When `/` is queried, the following is returned:
```
{
  "host_id": "e00c6a53124d06016a986d7a69989b88",
  "tagline": "The Disruptive Machine Learning API",
  "version": "1.4"
}
```
Can the minor version be included (in this case, `"version": "1.4.3"`)?

**Comment by daradermody** [comment]
Although Ubuntu (above) and Docker do not provide minor version, CentOS 7 does:
```
{"host_id":"8a677af9f5e02e5fd7fc65aa276c219e","tagline":"The Disruptive Machine Learning API","version":"1.4.3"}
```
@regel, can you clarify why this might be the case?

**Comment by regel** [comment]
+cc Christophe

Different builds, packaging scripts, using variables at build time to insert the version number. It seems only Docker packages have this issue. Have to check how we're building them.

**Comment by osunac** [comment]
The Docker image is built from the Debian packages.

We should be able to fix this in 1.4.4.
